package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "build": "yarn tslint && webpack --config webpack.config.js --display-error-details",
     "start": "node node_modules/http-server/bin/http-server -p 8765",
+    "start-sandbox": "tabextsandbox --config sandbox-config.json",
```

Comment:
this path doesn't seem right and assumes `tabextsanbox` is installed globally. maybe use `npx tabextsandbox`?